cis_enforcements/aws/cis_v300_s5_networking/network_acl.tf

```diff
@@ -0,0 +1,29 @@
+# AWS > VPC > Network ACL > Ingress Rules > Approved
+resource "turbot_policy_setting" "aws_vpc_network_acl_ingress_rules_approved" {
+  resource = turbot_smart_folder.aws_cis_v300_s5_networking.id
+  type     = "tmod:@turbot/aws-vpc-security#/policy/types/networkAclIngressRulesApproved"
+  value    = "Check: Approved"
+  # value    = "Enforce: Delete unapproved"
+  note     = "AWS CIS v3.0.0 - Controls: 5.1"
```

Comment:
order of the attributes should be:

1. resource
2. type
3. note
4. value
5. template input
6. template